tutorials/migrating_ardupilot_plugin.md

```diff
@@ -0,0 +1,926 @@
+\page ardupilot
+
+[TOC]
+
+# Case study: migrating the ArduPilot ModelPlugin from Classic Gazebo to Ignition Gazebo
+
+A variety of changes are required when migrating a plugin from Gazebo Classic
+("Gazebo") to Ignition Gazebo ("Ignition"). In this tutorial we offer as a case
+study the migration of one particular `ModelPlugin`,
+[ardupilot_gazebo](https://github.com/khancyr/ardupilot_gazebo). We hope that
+this example provides useful tips to others who are migrating their existing
+plugins from Gazebo to Ignition.
+
+The migrated version of the plugin that we're presenting here can be found in
+[this fork](https://github.com/gerkey/ardupilot_gazebo/tree/ignition). You can
+try building and running the migrated plugin in Ignition by following the
+[instructions in that
+fork](https://github.com/gerkey/ardupilot_gazebo/tree/ignition#using-with-ignition).
```

Comment:
```suggestion
The complete, migrated version of the `ardupilot_gazebo` plugin covered in this tutorial 
can be found in [this fork](https://github.com/gerkey/ardupilot_gazebo/tree/ignition).
```
The changed wording of the first sentence is just for extra clarity (not necessary), but I definitely think the link to the building-and-running instructions in the next lines shouldn't be here

Reply:
Thanks, @maryaB-osr !